pkg/volume/glusterfs/glusterfs.go

```diff
@@ -461,7 +461,7 @@ func getVolumeInfo(spec *volume.Spec) (string, bool, error) {
 
 	} else if spec.PersistentVolume != nil &&
 		spec.PersistentVolume.Spec.Glusterfs != nil {
-		return spec.PersistentVolume.Spec.Glusterfs.Path, spec.ReadOnly, nil
+		return spec.PersistentVolume.Spec.Glusterfs.Path, spec.PersistentVolume.Spec.Glusterfs.ReadOnly, nil
```

Comment:
All other volume plugins use `spec.ReadOnly`. I'd leave it as it is and solve it for all volume plugins in https://github.com/kubernetes/kubernetes/issues/70503

Reply:
@jsafrane thanks for pointing that open issue!. yeah, this is kind of a mess and I agree it should be solved for all plugins.